docs/source/getting-started/quickstart.adoc

```diff
@@ -49,50 +49,50 @@ $ minishift start
 To check the IP, run the `minishift ip` command.
 - By default, {project} uses the driver most relevant to the host OS.
 To use a different driver, set the `--vm-driver` flag in `minishift start`.
-For example, to use VirtualBox instead of KVM on GNU/Linux operating systems, run `minishift start --vm-driver=virtualbox`.
+For example, to use VirtualBox instead of KVM on Linux operating systems, run `minishift start --vm-driver=virtualbox`.
 - While {project} starts it runs several checks to make sure that the {project} VM and the OpenShift cluster are able to start correctly.
 If any startup checks fail, see the xref:../troubleshooting/troubleshooting-getting-started.adoc#minshift-startup-check-failed[Troubleshooting Getting Started] topic for information about possible causes and solutions.
 
 For more information about `minishift start` options, see the xref:../command-ref/minishift_start.adoc#[`minishift start`] command.
 ====
 
-. Use xref:../command-ref/minishift_oc-env.adoc#[`minishift oc-env`] to display the command you need to type into your shell in order to add the *oc* binary to your `PATH` environment variable.
+. Use xref:../command-ref/minishift_oc-env.adoc#[`minishift oc-env`] to display the command you need to type into your shell in order to add the `oc` binary to your `PATH` environment variable.
 The output of `oc-env` will differ depending on OS and shell type.
 +
 ----
 $ minishift oc-env
-export PATH="/Users/john/.minishift/cache/oc/v1.5.0:$PATH"
+export PATH="/home/john/.minishift/cache/oc/v1.5.0:$PATH"
```

Comment:
LOL? So you don't like macOS/BSD? ;-)

Reply:
"The following steps describe how to get started with Minishift on a **GNU/Linux** operating system with the KVM hypervisor driver." ;)